python/ray/dag/compiled_dag_node.py

```diff
@@ -63,13 +63,16 @@ def do_allocate_channel(
     self,
     reader_and_node_list: List[Tuple["ray.actor.ActorHandle", str]],
     typ: ChannelOutputType,
+    num_shm_buffers: int,
 ) -> ChannelInterface:
     """Generic actor method to allocate an output channel.
 
     Args:
         reader_and_node_list: A list of tuples, where each tuple contains a reader
             actor handle and the node ID where the actor is located.
         typ: The output type hint for the channel.
+        num_shm_buffers: The number of shared memory buffer per channel.
+            It is currently ignored for nccl channel.
```

Comment:
nit: remove "currently". 
If the variable is called `shm` buffers, then it should always not affect nccl.